db/migrate/20160909021628_training_log.rb

```diff
@@ -0,0 +1,16 @@
+class TrainingLog < ActiveRecord::Migration[5.0]
+  def up
+    create_table :training_logs do |t|
+      t.string :root_domain, unique: true, index: true
+      t.string :uri, index: true
```

Comment:
Actually, declaring `unique` here doesn't actually put a `unique` index constraint in the DB.

You'll have to do `add_index :training_logs, :uri, index: true`